dataplicity/tags.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+
+import subprocess
+import re
+import logging
+
+log = logging.getLogger("agent")
+
+
+TAG_SCRIPT = "/home/dataplicity/dataplicity.tags"
+
+
+class TagException(Exception):
+    pass
+
+
+def get_tag_list():
+    """Run the dataplicity.tags script, get output as a list of tags"""
+    try:
+        output = subprocess.check_output(TAG_SCRIPT)
+    except IOError:
```

Comment:
I think its `OSError` if the file doesn't exist, isn't it?

Reply:
Yes you're right.